saito-core/src/core/data/issuance

```diff
@@ -0,0 +1,4 @@
+5810123	n5hNA7Tazy7D226nq5arWq5YHxe8ibXGqNtjwXpEhnPT	VipOutput
```

Comment:
there are too many issuance files now. please move them to a single location

Reply:
The other issuance file is for testing purposes

Reply:
there's a saito-core/src/common/issuance also